examples/asr/transcribe_speech.py

```diff
@@ -113,6 +114,22 @@ class TranscriptionConfig:
     # General configs
     output_filename: Optional[str] = None
     batch_size: int = 32
+
+    beam_strategy: str = "beam"  # greedy, beam, flashlight, pyctcdecode
```

Comment:
Revert most of this. CTCDecoding is already part of transcribe speech config - https://github.com/NVIDIA/NeMo/blob/main/examples/asr/transcribe_speech.py#L138

Another thing is keep beam alpha and beta 1 and 0 respectively unless you know that the below values will always do better.

Reply:
Done